src/main/webapp/chart.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <script
+      type="text/javascript"
+      src="https://www.gstatic.com/charts/loader.js"
+    ></script>
+    //TODO move all of this to a separate JavaScript file
+    //figure out a way to get user input/data for the charts
+    //make sure to implement same idea as in the stats page
+    <script type="text/javascript">
+      google.charts.load("current", { packages: ["corechart"] });
+      google.charts.setOnLoadCallback(drawChart);
+      function drawChart() {
+        var book_data = new google.visualization.DataTable();
```

Comment:
Minor feedback: in JavaScript, it is more common to see `camelCase` than `lower_case_with_underscores` for variable names. See, for example:

https://google.github.io/styleguide/jsguide.html#naming-local-variable-names

I'm personally not a huge stickler for these conventions (as long as we are self-consistent), but I was instructed that Code U projects should uphold the Google style guides (I believe the purpose is to teach how to adhere to style guides as in a real software company's project, even when those style rules can sometimes be grating). Given that, I would appreciate it if you followed those conventions here.

Reply:
I blame the https://sites.google.com/codeustudents.com/spring-2019/weeks-3-7-guided-projects/charts-part-1 for not using camel case. :D 
Sounds good, I'll make sure to read through the Style Guide.